packages/next/client/image.tsx

```diff
@@ -16,6 +16,8 @@ import {
 } from '../shared/lib/image-config'
 import { ImageConfigContext } from '../shared/lib/image-config-context'
 import { warnOnce } from '../shared/lib/utils'
+// @ts-ignore - This is replaced by webpack alias
+import defaultLoader from 'next/image/loader'
```

Comment:
```suggestion
import defaultLoader from 'next/dist/shared/lib/image-loader'
```
This aliased path needs to match the fallback path so that it works without webpack applying the alias.